src/services/storage-options/restApiService.js

```diff
@@ -144,9 +142,12 @@ export default class RestApiService extends StorageInterface {
   }
 
   async getItemById(id) {
-    console.log(id);
-    const response = await axios.get(`http://localhost:8082/item`);
-    return response.data;
+    const itemInStore = store.state.session.items.find(
+      (item) => item.stepID === id
+    );
+    // const response = await axios.get(`http://localhost:8082/item`);
+    // console.log(response);
+    return itemInStore;
```

Comment:
Please remove commented out lines.

now that you have updated this method, please use it in the file above instead of directly accessing the state. @dockabernathy01

Reply:
Ok, I got it.